pkgs/applications/misc/minigalaxy/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib, fetchFromGitHub
+, docutils
+, gettext
+, glibcLocales
+, glib-networking
+, gobject-introspection
+, gtk3
+, python3
+, python3Packages
+, steam-run-native
+, unzip
+, webkitgtk
+, wrapGAppsHook,}:
```

Comment:
Could you format your code with [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt), this looks unfamiliar.

Reply:
Please also remove the superfluous lines 15, 29-31.

Reply:
thanks !